vision/style_transfer/onnx_helper.py

```diff
@@ -0,0 +1,106 @@
+import onnx
```

Comment:
where is this file from? the file should have a header with the license.

Also consider renaming it to something like pytorch2onnx_helper to be more clear on its purpose.

Reply:
I deleted this file.

Reply:
Great.